src/core/tsi/openssl_utils.cc

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright 2021 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include <absl/strings/string_view.h>
```

Comment:
nit: I think the usual format is openssl headers, blank line and then absl headers
Also shouldn't the openssl headers be in angled braces<> and string_view in ""

Reply:
Already changed it in https://github.com/grpc/grpc/pull/26690/commits/4631049ed89df6de589bd571a918eeb9d08a06f1

Reply:
I think you missed the comment on openssl headers. Prefer using angled braces.

Reply:
I definitely misunderstood you there. Also, clang_tidy doesn't seem to like when I use angled braces.
```
4251 warnings and 1 error generated.
Error while processing /local-code/src/core/lib/security/credentials/tls/grpc_tls_certificate_provider.cc.
/usr/local/google/home/itsemmanuel/.cache/bazel/_bazel_itsemmanuel/c59a765d249b50a9cc4dc379ce607f96/execroot/com_github_grpc_grpc/./src/core/tsi/openssl_utils.h:21:10: error: 'absl/strings/string_view.h' file not found with <angled> include; use "quotes" instead [clang-diagnostic-error]
#include <absl/strings/string_view.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
         "absl/strings/string_view.h"
Suppressed 4320 warnings (4251 in non-user code, 69 NOLINT).
Use -header-filter=.* to display errors from all non-system headers. Use -system-headers to display errors from system headers as well.
Found compiler error(s).
FAILED: src/core/lib/security/credentials/tls/grpc_tls_certificate_provider.cc [ret=1, pid=927, time=8.1sec]
```

Reply:
string_view shouldn't use angled brackets. I was talking about the openssl headers
```
#include <openssl/bio.h>
#include <openssl/evp.h>
#include <openssl/pem.h>
```

Reply:
Noted, thanks for your patience. :) It will be in the next push.